docs/json_rpc_api.mediawiki

```diff
@@ -1496,6 +1504,26 @@ of the best block.
 
 ----
 
+====getmixpairrequests====
+{|
+!Method
+|getmixpairrequests
+|-
+!Parameters
+|None
+|-
+!Description
+|Returns the current set of mixing pair request messages from the mixpool.
```

Comment:
only thing i would add is a message to not rely on this, it will get the chopping block soon. possibly replaced with a generic method to return the entire pool, or messages of a particular type, or whatever. but wallet won't be needing this one for too much longer

Reply:
Added:

>  WARNING: This is experimental and will very likely be removed in the next version.  Do not use!